docs/developer/add-new-source.md

```diff
@@ -13,10 +13,18 @@ What this does: This will update several files, including the addition of compon
 It is possible you may need to run this command twice. The first time will update the update process itself, and the 
 second time will run the update.
 
-
-## 2. `metadata/*.yaml`
+## 2. More config updates
```

Comment:
In addition to `mondo-ingest-odk.yaml`, configuration for new sources needs to be added in at least 3 other places, namely for `prefix_map`.

This redundancy will be addressed by:
- #146